test/units/TEST-82-SOFTREBOOT.sh

```diff
@@ -233,7 +236,7 @@ EOF
         --property After=basic.target \
         --property "Conflicts=reboot.target kexec.target poweroff.target halt.target emergency.target rescue.target" \
         --property "Before=reboot.target kexec.target poweroff.target halt.target emergency.target rescue.target" \
-        sleep infinity
+        bash -c 'count=0; while echo "$count"; do count=$[$count +1]; sleep 1; done'
```

Comment:
Could you check the continuation of the output after the soft-reboot?

Also, not a matter, but personally I prefer something like
```sh
for ((count=0; ; count++)); do echo "$count"; sleep 1; done
```

Reply:
if it cannot write because the fd is closed, the shell returns an error and terminates, so if the unit is still going, then it is writing

Reply:
Interesting, I did not know that. Nice.